hack/test-cmd.sh

```diff
@@ -264,4 +264,10 @@ openshift ex router --create --credentials="${KUBECONFIG}"
 [ "$(openshift ex router | grep 'service exists')" ]
 echo "ex router: ok"
 
+[ ! "$(openshift ex registry | grep 'does not exist')"]
```

Comment:
shouldn't this fail with some reasonable exit code instead of parsing the output?